xml/System.Globalization/CompareInfo.xml

```diff
@@ -1516,12 +1518,23 @@
         <Parameter Name="options" Type="System.Globalization.CompareOptions" Index="2" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">The text over which to compute the sort key.</param>
```

Comment:
>The text over which  [](start = 29, length = 20)

maybe say" the text to get the sortkey for? text over is kind of confusing I guess.

Reply:
```suggestion
        <param name="source">The text to get the sort key for.</param>
```